packages/web/src/download/Cover.tsx

```diff
@@ -9,6 +8,7 @@ import { I18nProps, NameSpaces, withNamespaces } from 'src/i18n'
 import { Cell, GridRow, Spans } from 'src/layout/GridRow'
 import { ScreenProps, ScreenSizes, withScreenSize } from 'src/layout/ScreenSize'
 import { standardStyles, textStyles } from 'src/styles'
+import { RequestType } from '../../server/FirebaseClient'
```

Comment:
Any reason we don't use an absolute import here like the others?

Reply:
there is an on going issue ive tried to fix where files in the /server folder can only be found relatively :(